weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -12,6 +12,9 @@ import {Query} from '../wfReactInterface/traceServerClientInterface/query';
 import {CallFilter} from '../wfReactInterface/wfDataModelHooksInterface';
 import {WFHighLevelCallFilter} from './callsTableFilter';
 
+// These operators do not require a value for the filter to be complete.
+const NO_VALUE_OPERATORS = ['(any): isEmpty', '(any): isNotEmpty'];
```

Comment:
might be nice to add this to `'../common/tabularListViews/operators'`

Reply:
Moved and tweaked a little.